### PR TITLE
Refine embed etcd metrics and logs

### LINKED ISSUE #16082: [Bug]: Remove etcd server metrics in milvus metrics
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:
- Deployment mode(standalone or cluster):
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

We've seen annoying metrics even if we didn't start embedded etcd server.

### Expected Behavior

We should remove these metrics if etcd server didn't start

### Steps To Reproduce

```markdown
package main

import (
	"context"
	"fmt"
	"log"
	"net/http"
	"time"

	"github.com/prometheus/client_golang/prometheus/promhttp"
	clientv3 "go.etcd.io/etcd/client/v3"
	"go.etcd.io/etcd/server/v3/embed"
)

var server *embed.Etcd

func main() {
	c, err := clientv3.New(clientv3.Config{
		Endpoints:   []string{"localhost:2379"},
		DialTimeout: 3 * time.Second,
	})

	if err != nil {
		log.Fatal("failed to connect to etcd: ", err.Error())
	}
	defer c.Close()
	http.Handle("/metrics", promhttp.Handler())
	http.HandleFunc("/", func(w http.ResponseWriter, req *http.Request) {
		ctx, cancel := context.WithTimeout(context.Background(), 2*time.Second)
		defer cancel()
		resp, err := c.Status(ctx, "localhost:2379")
		if err != nil {
			log.Fatal("failed get cluster status: ", err.Error())
		}
		fmt.Fprintf(w, "Leader: %v\n", resp.Leader)
		fmt.Fprintf(w, "Member id: %v\n", resp.Header.MemberId)

	})
	http.ListenAndServe(":2112", nil)
}
```

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by xiaofan-luan**
fix #16082
change etcd server log level to warn, if etcd server not started then no metrics should be collected 
Signed-off-by: xiaofan-luan <xiaofan.luan@zilliz.com>

### CODE REVIEW

File: configs/milvus.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ etcd:
     #  - "stdout" as os.Stdout,
     #  - file path to append server logs to.
     path: stdout
-    level: info # Only supports debug, info, warn, error, panic, or fatal. Default 'info'.
+    level: warn # Only supports debug, info, warn, error, panic, or fatal. Default 'info'.
```
Comment (soothing-rain) [inline configs/milvus.yaml:31]: nit: Default 'warn'.

Comment (xiaofan-luan) [reply configs/milvus.yaml:31]: Just thought the embed etcd log is a little bit too much, especially for standalone erver.
Maybe redirect to other directory?

Comment (soothing-rain) [reply configs/milvus.yaml:31]: Sure. I added a "path" setting above, so we can redirect logs to a specific directory. For example I use "/tmp/milvus/logs" as etcd log path in embedded Milvus.

File: configs/milvus.yaml [hunk 2]
```diff
@@ -49,7 +49,7 @@ pulsar:
   maxMessageSize: 5242880 # 5 * 1024 * 1024 Bytes, Maximum size of each message in pulsar.
 
 rocksmq:
-  path: /var/lib/milvus/rdb_data # The path where the message is stored in rocksmq
```
Comment (LoveEachDay) [inline configs/milvus.yaml:52]: Why change the data path for rocksmq?

Comment (xiaofan-luan) [reply configs/milvus.yaml]: that's just for test, remove it

File: configs/milvus.yaml (hunk 1 above)
Comment (xiaofan-luan) [reply configs/milvus.yaml]: sure, I've change the default back to info, it's a little bit of noisy

**Review by LoveEachDay — APPROVED** [review]
/lgtm

**Comment by LoveEachDay** [comment]
Fix #15751

**Pull request merged**